package.json

```diff
@@ -1,9 +1,10 @@
 {
   "name": "@stellar/typescript-wallet-sdk",
-  "version": "1.1.0",
+  "version": "1.1.1",
   "engines": {
     "node": ">=18"
   },
+  "browser": "./lib/bundle_browser.js",
   "main": "./lib/bundle.js",
```

Comment:
bundlers (like `webpack` and `rollup`) use the file specified in the `browser` field to build for web apps while all other apps default to the `main` field. 

So, we'll point to a specific browser build for web apps while defaulting to a node version

https://docs.npmjs.com/cli/v9/configuring-npm/package-json#browser